Connectors/Rhino/Speckle.Connectors.GrasshopperShared/HostApp/SpeckleResource.cs

```diff
@@ -6,15 +6,15 @@
 
 namespace Speckle.Connectors.GrasshopperShared.HostApp;
 
-public abstract record SpeckleUrlModelResource(string Server, string ProjectId)
+public abstract record SpeckleUrlModelResource(string? AccountId, string Server, string ProjectId)
```

Comment:
Add comment here, noting that if the user inputs a model url string, this will not contain account info (and that's why the accountID is nullable in the record resource)